### DISCUSSION TITLE
Maximum upload file size

### DISCUSSION THREAD

**Discussion by vinmaga** [outline/outline#4331] [General]
Hey guys, I'm using the free version of outline and I tried to upload a file larger than 262.14mb and get the message "sorry this file is too big - max size 262.14mb". I would like to understand if it's because I'm on the free version or this is the case with the paid version.

**Comment by ChetanGoti** [comment]
If you are using self-hosted version, you might want to configure `AWS_S3_UPLOAD_MAX_SIZE` to higher value

See https://github.com/outline/outline/discussions/4185#discussioncomment-3751166
